robottelo/ui/contentviews.py

```diff
@@ -55,18 +55,13 @@ def create(self, name, label=None, description=None, is_composite=False):
         self.wait_for_ajax()
         self.click(common_locators['create'])
 
-    def delete(self, name, really=True):
+    def delete(self, name):
```

Comment:
We have,but it is completely different and actually is not a message box, but a form inside of UI:
![image](https://cloud.githubusercontent.com/assets/10895065/11502962/a7eed10e-9846-11e5-998c-674c2f3a0f2b.png)

Basically, it is not mandatory to push cancel button as it is not a Javascript and you can proceed further and any action (which is not 'Delete' button) will withdraw delete operation. We can use, of course, custom search here and if necessary will do in future, but not now as we don't need to verify that button specifically (or we can do verification of cancel action separately)